content/applications/inventory_and_mrp/purchase/advanced/procurement_expenses_report.rst

```diff
@@ -0,0 +1,126 @@
+===========================
+Procurement expenses report
+===========================
+
+.. |RFQ| replace:: :abbr:`RfQ (Request for Quotation)`
+.. |RFQs| replace:: :abbr:`RfQs (Requests for Quotation)`
+.. |POs| replace:: :abbr:`POs (Purchase Orders)`
+.. |PO| replace:: :abbr:`PO (Purchase Order)`
+.. |caret| replace:: :icon:`fa-caret-down` :guilabel:`(down)` icon
+.. |per| replace:: procurement expenses report
+
+With the *Purchase* application, users can monitor procurement expenses over time. This report helps
+companies track and analyze spending, identify cost-saving opportunities, and ensure efficient
+budget management.
+
+Create procurement expenses report
+==================================
+
+To create a |per|, first navigate to :menuselection:`Purchase app --> Reporting --> Purchase` to
+open the :guilabel:`Purchase Analysis` dashboard. By default, the dashboard displays a line chart
+overview of the :guilabel:`Untaxed Total` of Purchase Orders (POs) with a :guilabel:`Confirmation
+Date` for the current month, or of Requests for Quotation (RFQs) with a status of *Draft*, *Sent*,
+or *Cancelled*.
+
+Add filters and groups
+----------------------
+
+On the top-right, click the :icon:`oi-view-pivot` :guilabel:`(pivot)` icon to switch to pivot view.
+
+.. tip::
+   While the |per| can also be :ref:`viewed <purchase/view-results>` as a :icon:`fa-bar-chart`
+   :guilabel:`(bar chart)`, :icon:`fa-line-chart` :guilabel:`(line chart)`, or :icon:`fa-pie-chart`
+   :guilabel:`(pie chart)`, the pivot view provides the most detailed view of the data, and is the
+   recommended starting point.
+
+Remove any default filters from the :guilabel:`Search...` bar. Then, click the |caret| to open the
+drop-down menu that contains the :guilabel:`Filters`, :guilabel:`Group By`, and
+:guilabel:`Favorites` columns.
+
+.. note::
+   Unless otherwise specified, the report displays data from both |RFQs| and |POs|. This can be
+   changed by selecting either :guilabel:`Requests for Quotation` or :guilabel:`Purchase Orders`
+   under the :guilabel:`Filters` column.
+
+Under the :guilabel:`Filters` column, select a date range to use for comparison. The report can be
+filtered by either :guilabel:`Order Date` or :guilabel:`Confirmation Date`. Choose one from the
+list, and click the |caret| to specify the date range, either by month, quarter, or year.
+
+Next, under the :guilabel:`Group by` column, select :guilabel:`Vendor`. Then, select
+:guilabel:`Product Category`, which is also located in the :guilabel:`Group By` column.
+
+.. note::
+   The selections under the :guilabel:`Group By` heading can be altered depending on the needs of
+   the individual company. For example, selecting :guilabel:`Product` instead of :guilabel:`Product
+   Category` provides a more in depth look at the performance of specific items, instead of an
+   entire category.
```

Comment:
```suggestion
   the individual company. For example, selecting :guilabel:`Product`, instead of :guilabel:`Product
   Category`, provides a more in depth look at the performance of specific items, in place of an
   entire category.
```